specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.5/TextAnalytics.json

```diff
@@ -2419,9 +2419,9 @@
       }
     },
     "StringIndexType": {
-      "default": "TextElements_v8",
+      "default": "TextElement_v8",
```

Comment:
interesting, is this a change the service is doing?

Reply:
we were actually supporting "TextElement_v8" even for previous versions but this was never an issue before because if a wrong value was sent we were mapping it to default value which was "TextElement_v8" which is why the SDK sending with an "s" was not causing an issue. From this version we will be doing a validation even for this param type

Reply:
that makes sense. Thanks for the explanation!